completion/functions/compstyle_ohmy_setup

```diff
@@ -0,0 +1,100 @@
+#!/bin/zsh
+
+### Oh-My-Zsh Project information:
+## Home:     https://ohmyz.sh
+## Authors:  Oh-My-Zsh contributers (https://github.com/ohmyzsh/ohmyzsh/graphs/contributors)
+## License:  MIT
+## Source:   https://github.com/ohmyzsh/ohmyzsh/blob/master/lib/completion.zsh
```

Comment:
I *think* this should probably refer to the current commit rather than `master` as it can change and it would be nice to know what version this was copied from.

Reply:
Changed to https://github.com/ohmyzsh/ohmyzsh/blob/97134d1eed19b72a9c4bf90a2effa847221608bf/lib/completion.zsh and pushed a new commit.